### DISCUSSION TITLE
Exception Error (Http.php:465)

### DISCUSSION THREAD

**Discussion by valentesolucoes** [wintercms/winter#1057] [Q&A] [answered]
Hello, I'm a beginner and I just installed Winter on my machine. Today, when trying to access the plugin installation screen (Settings > System > Updates & Plugins > + Install plugins) the following error appeared:

TypeError: Winter\Storm\Network\Http::data(): Argument #2 ($value) must be of type ?string, array given, called in /Volumes/www/domains/winter/vendor/winter/storm/src/Network/Http.php on line 469 and defined in /Volumes/www/domains/winter/vendor/winter/storm/src/Network/Http.php:465

**Comment by AIC-BV** [comment, marked as answer]
Hi Valentesolucoes
I don't know that error straight away, but installing plugins have to happen through composer (for now) untill the Marketplace is live.

Example:
`composer require winter/wn-blog-plugin` in command line
& then `php artisan winter:up` to run the migrations

Hopefuly this can get you started untill someone can help you with this error

**Reply by valentesolucoes** [reply]
Thank you very much !!

**Comment by damsfx** [comment]
While we're waiting for the marketplace to open, I'm wondering whether an information page shouldn't replace this controller.  
We often get the same errors when users try to install plugins and themes via the GUI.

**Reply by AIC-BV** [reply]
Yes it is a common question among new winter users.
It should be more clear for people how to install plugins in the mean time.

**Comment by mjauvin** [comment]
@damsfx that error is caused by a bug I introduced in 1.2.4

It has been fixed in latest develop branch and will be available in 1.2.5
